### DISCUSSION TITLE
RustDesk iOS - Remember or configure different default display resolution per device

### DISCUSSION THREAD

**Discussion by plia7** [rustdesk/rustdesk#9781] [General]
Hello,

When I connect to an Android target using RustDesk iOS app (version 1.1.0), I need to click the monitor icon in the blue menu below to choose from the following display resolution settings (The default selected is Shrink):
- Original
- Shrink
- Stretch

I have to change it each time to "Original" so the target is more accessible/clickable in the connecting screen after connecting.
Is there a way to make it remember the last display resolution setting that was selected?
If not, then this could be a nice feature to add to configure it ahead of time per device or at least remember the last selection.

Thank you.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/pull/9786
